apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/VedtaksvurderingRepository.kt

```diff
@@ -107,7 +106,7 @@ class VedtaksvurderingRepository(datasource: DataSource) {
                 Beregningsresultat::class.java
             )
         },
-        vilkaarsResultat = stringOrNull("vilkaarsresultat")?.toJsonNode(),
+        vilkaarsResultat = stringOrNull("vilkaarsresultat")?.let { objectMapper.readTree(it) },
```

Comment:
Denne kan endres til å mappes til vilkårsvurderingsklassen hvis vi wiper dbene

Reply:
> Denne kan endres til å mappes til vilkårsvurderingsklassen hvis vi wiper dbene, slik som beregning hentes ut.

On it! https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/492

Reply:
@henrikg @simejo Var tanken å ha beregning og vilkårsvurdering som jsonnode slik at man kan ha flere versjoner? Eller er det en sak for postkassa?
Det ble jo innført bare så man kunne få migrert vilkårsdatabasen for noen mnd siden.

Reply:
Tanken med all dataen som lagres i Vedtak var vel kun for dokumentasjon. At man valgte jsonnode bare for å dumpe ned data som man senere kan lese raw uten noen klasse (som skal være godt nok for etterlevelseskrav). Det gjelder for så vidt vilkårsvurderingen også